Clicker/ViewControllers/EditNameViewController.swift

```diff
@@ -88,15 +89,24 @@ class EditNameViewController: UIViewController {
         }
     }
     
+    func updateSession(id: Int, name: String, code: String) -> Future<Response<Node<Session>>> {
+        return networking(Endpoint.updateSession(id: id, name: name, code: code)).decode(Response<Node<Session>>.self)
```

Comment:
I think if you have the return type as `Future<Response<Node<Session>>>`, you can just do `return ...decode()`. @AAAstorga can you confirm?